todo-backend-typescript/package.json

```diff
@@ -4,17 +4,15 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "link": "npm link featurehub-repository && npm link featurehub-eventsource",
     "build": "node_modules/.bin/tsc",
-    "compile": "npm run link && npm run build && cp package.json dist && cd dist",
-    "start": "npm run link && npm run build && node dist/app.js"
+    "compile": "npm run build && cp package.json dist && cd dist",
```

Comment:
do you know what the purpose of this line is?

Reply:
no? is it not needed?

Reply:
no, i don't think so, it is weird.